CMakeLists.txt

```diff
@@ -784,6 +785,8 @@ set(TESSERACT_CONFIGS
     tessdata/configs/lstmbox
     tessdata/configs/lstmdebug
     tessdata/configs/makebox
+    tessdata/configs/page
+    tessdata/configs/page_poly
```

Comment:
I suggest to install only `page` and modify it to use polygons instead of bounding boxes.